en/micro-integrator/docs/setup/user_stores/setting_up_ro_ldap.md

```diff
@@ -40,6 +40,12 @@ Follow the steps given below to connect the Micro Integrator to the LDAP user st
 	connection_password = "admin"  
 	user_search_base = "ou=system"   
 	```
+	
+	!!! Note
+		If you have taken a WUM update after 28/11/2019, you need to add the following configuration under `[user_store]`.
```

Comment:
this config is in addition to the above config ?

Reply:
yes. addition to the above